### DISCUSSION TITLE
collective decision-makingをProject Sprintの記述に取り込めないか

### DISCUSSION THREAD

**Discussion by KokoroKagawa** [copilot-jp/project-sprint#386] [Development]
先日@yukikomoriyaさんに調べていただいたcollective decision-makingに関して、Project Sprintの記述に取り込みたい！

方法論も将来的に記述に活かせそうだが、まずは「なぜ集合的意思決定が重要なのか」の部分をもう少し掘り下げたい。
Project Sprintでは、チームが自律的・漸進的に成果を生み出し続けるために、「チームとしての納得を持ってゴールを設定する」「個々のメンバーの作業結果を持ち寄って目線を合わせ、チームとして次のアクションのための意思決定をする」ということを大事にしている。
この価値観の重要性をcollective decision-makingの周辺の考え方が裏付けてくれないだろうか、という視点での深掘り希望。

**Comment by motoi** [comment]
https://github.com/copilot-jp/project-sprint/issues/244
同様の議論をしていたので参考までに経緯としてはります

**Comment by yukikomoriya** [comment]
### [最新 心理学事典]「集団意思決定」の解説
https://kotobank.jp/word/%E9%9B%86%E5%9B%A3%E6%84%8F%E6%80%9D%E6%B1%BA%E5%AE%9A-2099788

> 人間社会に集団意思決定が広く浸透している理由としては
> 
> 1. 民主的な民意を反映した決定を行なえること
> 2. 個人で判断するよりも的確な判断が行なえること
> 3. 個人で考えるよりも創造的なアイデアを思いつきやすくなること
> 
> などが期待できることにあると考えられる。
> 
> ただ、合議の手続き次第で決定を操作しうることを示す研究が報告されており（亀田達也1997）、集団意思決定は民意を的確に反映したものとはならない可能性を多く含むことが指摘されている。また、集団意思決定の的確性や創造性に関してもこれらの期待を叶えることは難しいことを示す研究結果が多数報告されている。集団意思決定の的確性を阻害するものとしてはグループシンクや集団極化がよく知られているし、ブレーンストーミングに関する研究は集団意思決定による創造的アイデア生成の難しさを示すものとなっている。より優れた集団意思決定を行なうために、これらの困難要因を克服する取り組みが重要課題であり、コンピュータの発展とともに研究知見を生かしたグループウエアの検討と開発が行なわれてきている。

--小森谷コメント
集合的意思決定の「WHY：なぜ必要か」としては上記３点が提示されているものの、HOWに課題がありそうな感じですね🤔
もう少し調べてみます！

**Comment by yukikomoriya** [comment]
本日のmtgにて、
意思決定に繋がる「議論の収束」には以下２つの流れが同時に存在するのではないか？
というお話をさせていただきました。

**1. ロジカル（論理的・合理的）な収束
2. エモーショナル（感情的）な収束**

「感情的な収束」について考えると、個人的には議論よりも対話によって収束していくイメージがあります。
たとえば以下のスクショはU理論のオットーシャーマー・モデルですが、
儀礼的対話から討論と内省・共感を経て共創的対話に辿り着くまでに、何か感情的なもの？を手放していくイメージです。

<img width="657" alt="スクリーンショット 2022-08-30 17 49 11" src="https://user-images.githubusercontent.com/111475379/187398260-3a84486a-45e2-4686-9eb9-34b3be4e8dc8.png">


ちなにみ上記スクショは以下記事からの引用です。
[『社会変革のためのシステム思考実践ガイド』の監訳者による解説を全文公開します。（EIJI Books）](https://eijionline.com/n/n4d53e5328b08)

「コレクティブ・ディシジョン・メイキング」ではなく「コレクティブインパクト」という言い方をしていますが、
書籍の目次を見ると今日の話と関連しそうな内容かも、と思いました。
もしすでに読んだ方いらっしゃいましたら感想などお伺いしたいです！

**Comment by KokoroKagawa** [comment]
元の書籍未読なので雑感レベルですが。。。

エモーショナルな収束に向けた儀礼的会話⇒討論⇒内省的対話⇒共創的対話は、個々人が自己を変容させ、チームとしてのナラティブ的なものを獲得・統合・合意していく過程ともいえそう。感情を手放して論理に落とし込むというよりは、感情も含めて論理を統合していくイメージの方がしっくりきます。
感情が収束していなくてもdisagree and commit状態で進むことは可能。
ロジカルな収束の結論に納得できるということがエモーショナルな収束？

挙げていただいたスクショの前の↓の記述は、まさにPJSにおけるミーティングの「場を固定する」という機能だなあと思いました。
>囲われた場に蓋があることで、器の中でエネルギーや感情が発散され、ぶつかり、やがて気圧（プレッシャー）が高まる中で調和していく一つの大きな流れが形成されるように器を保持できる機能が形成されます

定金さんが話題に出していた「ラフコンセンサス、ランニングコード」の記事も貼っておきます。
https://gigazine.net/news/20190613-rough-consensus/

**Comment by yukikomoriya** [comment]
「なぜ集合的意思決定が重要なのか」について、人文ではなく数理的な観点も調べてみました。

### 陪審の定理（コンドルセの定理）
集団の意思決定を機能させるための6つの前提条件

1.集団内の個人は独立していなければならない
2.集団内の個人は互いの意見に影響されてはならない
3.偏りがあってはならない
4.全員が同じ問題に答えようとしなければならない
5.正しい答えを導くのに必要な情報がなければならない
6.問題には正解がなければならない

世界で最もイノベーティブな組織の作り方 (光文社新書)  -  山口 周
https://amzn.to/3cBy1Gi



### コンドルセの定理（wikipedia）
https://ja.wikipedia.org/wiki/%E9%99%AA%E5%AF%A9%E5%AE%9A%E7%90%86
> 社会契約論に於いてルソーは、国家の主権の根拠となる一般意志の概念を提唱した。しかしルソーは、多数決によって得られる意志を「全体意志」として一般意志と区別し、国家にとって絶対の真理となる一般意志を決定する手続きについては明確にしなかった。
> そこでコンドルセは、政治思想とは無関係の理論体系である確率論を用いる事で、多数決の信頼性を定量的に見積もろうとした。

### コンドルセの陪審定理
https://www.geogebra.org/m/hf42kzxp
> コンドルセの陪審定理のカラクリとは、
> 個人が正しい選択をする確率が５割以上なら、平均すれば過半数の人が正しい選択をするが、確率的なゆらぎによって過半数を下回ることも結構ある。しかし、投票者の数が増えると、大数の法則が働いてゆらぎが小さくなり、ほぼ確実に過半数を超えるようになる。

<img width="576" alt="スクリーンショット 2022-09-01 19 42 11" src="https://user-images.githubusercontent.com/111475379/187895687-64f33e2d-5afe-4545-baee-3fab463db034.png">

> **うまくいかないケース**
> - 運が悪い
> - 選択肢がない
> - 利己的な選択をしてしまう
> - 投票者の数が足りない
> - 独立性が足りない


### おまけ：コンドルセのパラドックス
https://imidas.jp/genre/detail/A-101-0133.html
> 個人が選択肢に対して首尾一貫した好みを持っているにもかかわらず、多数決によって集団全体の好みを決定しようとすると、選択肢間ではっきりと優劣が付かなくなることを、投票のパラドックスと言う。よりフォーマルには、推移性（transitivity）を満たす個人の選好から、多数決によって導かれた社会的な選好への推移性を満たさなくなること、を指す。

**Reply by KokoroKagawa** [reply]
コンドルセの定理、1～5までは納得感あったけど、「6.問題には正解がなければならない」はどうかなあ。
アジャイル型のプロジェクトに「正解」はあるのか？
そもそも正解がない世界の中で、集団的意思決定の結果が「正解」となるよう行動していくのか？

…と思ったら書いてありました。
>未来の予測を含む不可知な条件下での意思形成には必ずしも適用できない
>選択肢が三つ以上の問題については触れていない

**Comment by yukikomoriya** [comment]
### 意思決定とは？よい意思決定を行うためのポイントを解説
https://www.sofia-inc.com/blog/9370.html

ハーバート・サイモンの意思決定論、アンゾフの意思決定論、をベースに、
「コントロールできる要素における意思決定」と「コントロールできない要素における意思決定」に分類して考える記事。

**コントロールできる要素における意思決定**
明確な目的と、判断の根拠となる過去のデータや事実をもとに行う意思決定
①目的・問題の特定
②情報収集と共有
③合意形成

**コントロールできない要素における意思決定**
判断の根拠となるようなデータや事実が存在しない問題に対して行う意思決定
①ビジョン
②より多くの選択肢を生み出す仕組み
③化学反応を生み出す場

**まとめ**
> 明確な前提のある中で意思決定を行う際には、押さえるべきポイントがあり、良い意思決定ができるかどうかは技術的な問題です。一方で、不確実な未来に関わる意思決定を行う際は、選択肢を自分たちで生み出し、自分たちで選び取るというプロセス自体が組織にとって重要です。こういったビジョンやプロセスを提示するのは、マネジメントの大切な役割です。
> 
> コロンビア大学ビジネススクール教授のシーナ・アイエンガー氏は、その著書『選択の科学』の最終章において「選択の力を最大限に活用するには、その不確実性と矛盾を受け入れなくてならないのだ。」と述べています。より良い意思決定で未来を切りひらくためには、問題に直面している当事者自身が、現実の不確実性と矛盾を受け入れるマインドセットを行い、その状況でどのように前に進むか考えることこそが重要なのです。

--小森谷メモ
『選択の科学』、積読状態なので読もうと思います...！

**Comment by yukikomoriya** [comment]
集 団 意 思 決 定 と コ ミ ュ ニ ケ ー シ ョ ン ・モ ー ド1)
― コ ン ピ ュ ー タ ・コ ミ ュ ニ ケ ー シ ョ ン条 件 と対 面 コ ミ ュ ニ ケ ー シ ョ ン条 件 の
差 異 に 関 す る 実 験 社 会 心 理 学 的 検 討 ―
https://web.archive.org/web/20181102074607id_/https://www.jstage.jst.go.jp/article/jjesp1971/38/2/38_2_183/_pdf

オフラインの対面コミュニケーションと、オンラインのテキストコミュニケーションの差が
集団意思決定に与える影響についての論文です。
オンラインのテキストコミュニケーションの場合、オフラインでは機能する対人圧力が希薄化するので
よりリスキーな選択をしがちとのこと。
これは、集団意思決定が「リスク回避」の機能を持ってることの裏返しかと思います。

リスク回避自体はメリットにもデメリットにも働くので、
「なぜ集合的意思決定が重要なのか」の答えにはならないな、と思いつつつ
リモート前提の意思決定のあり方を考える際の参考までに、投稿させていただきます。

実際に私自身もリモートシフトで「対人圧力の希薄化」を実感しているので、興味深いです。
